manifests/c/CTerm/CTerm/3.7.0/CTerm.CTerm.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: CTerm.CTerm
+PackageVersion: 3.7.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+ProductCode: CTerm_is1
+ReleaseDate: 2025-01-15
+Installers:
+- Architecture: x86
+  InstallerUrl: https://gitee.com/devcpp/cterm/releases/download/v3.7.0/CTerm-3.7.0.exe
+  InstallerSha256: 08EB3ED56CAC6F0D0D5DEC7E000294ADE5742A77112064CC5EB4735EB24835BA

```

```suggestion
 InstallerSha256: 3AE7229F19A6B3B949087EBB322B725BB9DF47F4510BCC57D99A50C8D69F3666
```

(Automated response - build 949.)